Cmdr/BuiltInCommands/Utility/runif.lua

```diff
@@ -42,9 +102,9 @@ return {
 			return ("Condition %q is not valid."):format(condition)
 		end
 
-		local text = conditionFunc(testAgainst, arg)
+		local success, text = pcall(conditionFunc, testAgainst, arg)
 
-		if text then
+		if success and text then
```

Comment:
To support the fact that the command arg is optional, if the user just wants to run a runif conditional test and see what the result is without actually running a specific command we can change the line 107 block to this:
```lua
if success then
	if command and text then
		return context.Dispatcher:EvaluateAndRun(
			context.Cmdr.Util.RunEmbeddedCommands(context.Dispatcher, command or text)
		)
	elseif not command then
		return ("Condition %q returned %s."):format(condition, text and "true" or "false")
	end
end